src/components/selection_input/selection_input.ts

```diff
@@ -170,6 +183,9 @@ export class SelectionInput extends Component<Props, SpreadsheetChildEnv> {
         ev.preventDefault();
         updateSelectionWithArrowKeys(ev, this.env.model.selection);
       }
+    } else if (ev.key === "Enter") {
+      const target = ev.target as HTMLInputElement;
+      target.blur();
```

Comment:
@rrahir It's indeed useless, and it's added for user experience, otherwise when users hit enter the input cursor it still there (which I feel weird). But it can be deleted anyway 🙂

Reply:
> it's added for user experience

It's not useless then :sweat_smile: 
That being said, the behavior of the Enter key could still be improved IMO. I think it should have the same effect as hitting the Confirm button